api/db/changelog/db.changelog-169-add-institution-tier-requirement-tables.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="changelog-169-add-institution-tier-requirement-tables">
+    <createTable tableName="institution_tier_requirement">
+      <column name="institution_tier_requirement_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="requirement_enum" type="ENUM('DOMAIN', 'ACL', 'NONE')">
+        <constraints nullable="false"/>
+      </column>
+      <column name="era_required" type="boolean">
+        <constraints nullable="false"/>
+      </column>
+      <column name="institution_id" type="bigint">
```

Comment:
for mapping tables like these, I prefer to have these fields as the next ones after the PK so it's obvious what is being mapped